srcts/src/components/sidebar.ts

```diff
@@ -314,7 +314,10 @@ class Sidebar {
       .getPropertyValue("--bslib-sidebar-js-init-collapsed");
 
     if (initCollapsed.trim() === "true") {
-      this.toggle("close");
+      container.classList.add(Sidebar.classes.COLLAPSE);
+      this._finalizeState();
+    } else {
+      sidebar.hidden = false;
     }
   }
```

Comment:
```suggestion
      container.classList.add(Sidebar.classes.COLLAPSE);
    }
    this._finalizeState();
```

Reply:
More generally, this is reaching into the internal logic of `toggle()` and it'd be nice to have this become something like `toggle("close", immediate = true)`

Reply:
Done in https://github.com/rstudio/bslib/pull/907/commits/79de52212152619753bdaf7a4d5224dd80efe35d, I moved all logic into the `.toggle()` method, which is now called with the appropriate `method`, i.e. for both `"open"` and `"close"`.